lib/Refactoring/Refactoring.cpp

```diff
@@ -595,9 +595,7 @@ class TextReplacementsRenamer : public Renamer {
     assert(Old.partsCount() == New.partsCount());
   }
 
-  std::vector<Replacement> getReplacements() const {
-    return std::move(Replacements);
-  }
+  ArrayRef<Replacement> getReplacements() const { return Replacements; }
```

Comment:
Really not sure why we were returning `std::vector`  while the all usages expect `ArrayRef` 🤔

Reply:
Me neither :)! Especially since we were moving it out and thus `Replacements` would not longer be valid 🤷.